README.md

```diff
@@ -88,17 +91,26 @@ Color: whitesmoke (#F5F5F5)
 - `in progress`: Someone is working on it.  
 - `sprint ready`: Ready to estimate.  
 
+**Close labels**  
+Color: white (#FFFFFF)  
+Close the issue when assigning one of these.  
+- `wont fix`: This will not be fixed for an arbitrary reason (e.g. a crash in a part of the App that will be removed soon).  
+Before closing, provide the reason as comment if possible.  
+- `duplicate`: Title says it all. Always keep the issue with the most information.  
+Before closing, add comment ```closed as duplicate of #<issue-that-is-kept>``` to remove the need of migrating the information to the kept issue.  
```

Comment:
I guess that's all of them, at least I can't think of any other close reasons.
We have 'duplicate', 'wontfix', 'merged'. Technically, we maybe want to migrate 'merged' from the special labels to the close labels?